### DISCUSSION TITLE
Server-wide default rate provider for new stores/users

### DISCUSSION THREAD

**Discussion by jacobburrell** [btcpayserver/btcpayserver#5382] [Ideas / Feature Requests]
Several users have encountered issues after creating an account on BTCPayServer. Specifically, they run into problems when trying to select an exchange or trading pair that isn't supported by the platform. The majority of these users aren't initially concerned with choosing a specific exchange, yet this issue tarnishes their first experience.

The problem doesn't manifest at the invoice or payment request stage, but rather when a client attempts to complete a payment—resulting in a failure. To mitigate this, could we introduce a server-wide default setting for new users? Ideally, this would be an advanced option that users can later customize to suit the needs of any store they set up.

**Comment by Kukks** [comment]
Can you provide some concrete examples

**Comment by dennisreimann** [comment]
>  The majority of these users aren't initially concerned with choosing a specific exchange

That's why we offer sane defaults, which should work in general.

> yet this issue tarnishes their first experience

What do they do and set, so that it does not work?

**Comment by pavlenex** [comment]
I would assume on the initial startup, the user selects USD as default but ends up using something completely different, RSD for example?

I think I encountered this issue already, I think what @Kukks mentioned is that we should have some sort of fallback or defaults in case an invoice can't be generated with a default rate provider fallback to a default source for that currency?

I also am unsure if this is what you're describing, it seems you're actually talking about efforts when making a payment on the client side.
